custom_components/localtuya/config_flow.py

```diff
@@ -44,6 +44,7 @@
 from .coordinator import pytuya, TuyaCloudApi
 from .core.cloud_api import TUYA_ENDPOINTS
 from .core.helpers import templates, get_gateway_by_deviceid, gen_localtuya_entities
+from .core.ha_entities.base import DPCode
```

Comment:
no need to make exceptions for DPS instead remove the current condition for now. If there are DPS without value and can be used then the condition is not always accurate.